contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcViews.java

```diff
@@ -0,0 +1,143 @@
+          RpcViewConstants.RPC_SERVER_UNCOMPRESSED_REQUEST_BYTES_VIEW,
+          RpcViewConstants.RPC_SERVER_UNCOMPRESSED_RESPONSE_BYTES_VIEW);
+
+  @VisibleForTesting
+  static final ImmutableSet<View> RPC_INTERVAL_VIEWS_SET =
+      ImmutableSet.of(
+          RpcViewConstants.RPC_CLIENT_ERROR_COUNT_MINUTE_VIEW,
+          RpcViewConstants.RPC_CLIENT_ROUNDTRIP_LATENCY_MINUTE_VIEW,
+          RpcViewConstants.RPC_CLIENT_REQUEST_BYTES_MINUTE_VIEW,
+          RpcViewConstants.RPC_CLIENT_RESPONSE_BYTES_MINUTE_VIEW,
+          RpcViewConstants.RPC_CLIENT_REQUEST_COUNT_MINUTE_VIEW,
+          RpcViewConstants.RPC_CLIENT_RESPONSE_COUNT_MINUTE_VIEW,
+          RpcViewConstants.RPC_CLIENT_UNCOMPRESSED_REQUEST_BYTES_MINUTE_VIEW,
+          RpcViewConstants.RPC_CLIENT_UNCOMPRESSED_RESPONSE_BYTES_MINUTE_VIEW,
+          RpcViewConstants.RPC_CLIENT_SERVER_ELAPSED_TIME_MINUTE_VIEW,
+          RpcViewConstants.RPC_SERVER_ERROR_COUNT_MINUTE_VIEW,
+          RpcViewConstants.RPC_SERVER_SERVER_LATENCY_MINUTE_VIEW,
+          RpcViewConstants.RPC_SERVER_SERVER_ELAPSED_TIME_MINUTE_VIEW,
+          RpcViewConstants.RPC_SERVER_REQUEST_BYTES_MINUTE_VIEW,
+          RpcViewConstants.RPC_SERVER_RESPONSE_BYTES_MINUTE_VIEW,
+          RpcViewConstants.RPC_SERVER_REQUEST_COUNT_MINUTE_VIEW,
+          RpcViewConstants.RPC_SERVER_RESPONSE_COUNT_MINUTE_VIEW,
+          RpcViewConstants.RPC_SERVER_UNCOMPRESSED_REQUEST_BYTES_MINUTE_VIEW,
+          RpcViewConstants.RPC_SERVER_UNCOMPRESSED_RESPONSE_BYTES_MINUTE_VIEW,
+          RpcViewConstants.RPC_CLIENT_ERROR_COUNT_HOUR_VIEW,
+          RpcViewConstants.RPC_CLIENT_ROUNDTRIP_LATENCY_HOUR_VIEW,
+          RpcViewConstants.RPC_CLIENT_REQUEST_BYTES_HOUR_VIEW,
+          RpcViewConstants.RPC_CLIENT_RESPONSE_BYTES_HOUR_VIEW,
+          RpcViewConstants.RPC_CLIENT_REQUEST_COUNT_HOUR_VIEW,
+          RpcViewConstants.RPC_CLIENT_RESPONSE_COUNT_HOUR_VIEW,
+          RpcViewConstants.RPC_CLIENT_UNCOMPRESSED_REQUEST_BYTES_HOUR_VIEW,
+          RpcViewConstants.RPC_CLIENT_UNCOMPRESSED_RESPONSE_BYTES_HOUR_VIEW,
+          RpcViewConstants.RPC_CLIENT_SERVER_ELAPSED_TIME_HOUR_VIEW,
+          RpcViewConstants.RPC_SERVER_ERROR_COUNT_HOUR_VIEW,
+          RpcViewConstants.RPC_SERVER_SERVER_LATENCY_HOUR_VIEW,
+          RpcViewConstants.RPC_SERVER_SERVER_ELAPSED_TIME_HOUR_VIEW,
+          RpcViewConstants.RPC_SERVER_REQUEST_BYTES_HOUR_VIEW,
+          RpcViewConstants.RPC_SERVER_RESPONSE_BYTES_HOUR_VIEW,
+          RpcViewConstants.RPC_SERVER_REQUEST_COUNT_HOUR_VIEW,
+          RpcViewConstants.RPC_SERVER_RESPONSE_COUNT_HOUR_VIEW,
+          RpcViewConstants.RPC_SERVER_UNCOMPRESSED_REQUEST_BYTES_HOUR_VIEW,
+          RpcViewConstants.RPC_SERVER_UNCOMPRESSED_RESPONSE_BYTES_HOUR_VIEW);
+
+  /**
+   * Registers all standard cumulative views.
+   *
+   * <p>This must be called once at the beginning of the process.
+   *
+   * @since 0.11.0
+   */
+  public static void registerCumulativeViews() {
```

Comment:
I think `registerAllCumulativeViews()` and `registerAllIntervalViews()` would be more consistent with `registerAllViews()`.

Reply:
Done.